modules/core/src/v2/coins/eos.ts

```diff
@@ -458,17 +484,20 @@ export class Eos extends BaseCoin {
   }: ExplainTransactionOptions): Bluebird<DeserializedEosTransaction> {
     const self = this;
     return co<DeserializedEosTransaction>(function* () {
-      const eosClientConfig = {
+      // create an eosjs API client
+      const api = new Api({
+        abiProvider: new OfflineAbiProvider(),
         chainId: self.getChainId(),
-        transactionHeaders: headers,
-      };
-      const eosClient = new EosJs(eosClientConfig);
+        textDecoder: new TextDecoder(),
+        textEncoder: new TextEncoder(),
+      });
 
-      // Get tx base values
-      const eosTxStruct = eosClient.fc.structs.transaction;
       const serializedTxBuffer = Buffer.from(transaction.packed_trx, 'hex');
-      const tx = EosJs.modules.Fcbuffer.fromBuffer(eosTxStruct, serializedTxBuffer);
-
+      const deserializedTxJsonFromPackedTrx = yield api.deserializeTransactionWithActions(serializedTxBuffer);
```

Comment:
`deserializeTransactionWithActions` expects a String or Uint8Array as input. Passing in a buffer will throw an error.

confirmed this will work by passing in `packed_trx` directly.
`const deserializedTxJsonFromPackedTrx = yield api.deserializeTransactionWithActions(transaction.packed_trx);`

Reply:
is still an issue?